test_data/cloud-init/clear/openstack/latest/user_data

```diff
@@ -63,5 +63,6 @@ write_files:
         bash -c "echo 1af4 1041 > /sys/bus/pci/drivers/vfio-pci/new_id"
         bash -c "echo 0000:00:06.0 > /sys/bus/pci/devices/0000\:00\:06.0/driver/unbind"
         bash -c "echo 1af4 1041 > /sys/bus/pci/drivers/vfio-pci/new_id"
-
-        /mnt/cloud-hypervisor --kernel /mnt/vmlinux --cmdline "console=hvc0 reboot=k panic=1 nomodules i8042.noaux i8042.nomux i8042.nopnp i8042.dumbkbd root=/dev/vda2 VFIOTAG" --disk path=/mnt/clear-31310-cloudguest.img path=/mnt/cloudinit.img --cpus boot=1 --memory size=512M --device path=/sys/bus/pci/devices/0000:00:05.0/ path=/sys/bus/pci/devices/0000:00:06.0/
+        echo 4096 | sudo tee /proc/sys/vm/nr_hugepages
```

Comment:
You could have also done it differently, by allowing the amount of pages you want from the L1 kernel command line with `hugepagesz=2M hugepages=256` since you know L2 is gonna be booted with 512M of RAM.
Not sure what's the most canonical way.

Reply:
I like that it's the same method for L1 so i'd rather not change it.

Reply:
Ok fine by me, but at least change the number because it doesn't make sense to create 4096 hugepages when the L1 VM is only 1G, right?

Reply:
Is it just the RAM for the guest? I though that there might be other pages such as those associated with the pass throughed devices?

Reply:
Oh maybe, but the performance improvement is really about having hugepages for the RAM, because it will accelerate the update of the IOMMU through the vIOMMU. That's mainly the reason why hugepages improves things in this context.